src/main/java/org/fundacionjala/coding/wilson/CC.java

```diff
@@ -0,0 +1,32 @@
+package org.fundacionjala.coding.wilson;
+
+/**
+ *
+ */
+public final class CC {
+    /**
+     *
+     */
+
+    private CC() {
+
+    }
+
+    /**
+     *
+     * @param str is a sentence
+     * @param c is a model
+     * @return cant c in the sentence
+     */
+    public static int charCount(final String str, final char c) {
+        int sum = 0;
+        String str1 = str;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "str1". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "str1" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)